src/tpm2-tss-engine-ecc.c

```diff
@@ -595,6 +694,7 @@ init_ecc(ENGINE *e)
         = NULL;
     EC_KEY_METHOD_get_sign(ecc_methods, &orig_sign, NULL, NULL);
     EC_KEY_METHOD_set_sign(ecc_methods, orig_sign, NULL, ecdsa_sign);
+    EC_KEY_METHOD_set_compute_key(ecc_methods, ecdh_compute_key);
```

Comment:
I think this needs to be guarded by an #ifdef as well, right ?

Reply:
This is defined within `#else /* OPENSSL_VERSION_NUMBER < 0x10100000 */` on line 697, which I think is sufficient. Also, the tests pass which build for all supported versions of OpenSSL, so I think it's good as-is. Though, I defer to your experience in the codebase here.

Reply:
Oh, sorry. I missed that.
Cool... I'll merge it !